ProjectTaskTracker.js

```diff
@@ -1,77 +1,152 @@
 
+/*-------------------------------creating the template-------------------------------------------*/
+
 const addNewProjectButton = document.getElementsByClassName("addButton")[0];
 const addTaskButton = document.getElementsByClassName("addButton")[1];
-var toDoTaskNumber = 0;
-var projectNumber = 0;
+
+const deleteProjectButton = document.getElementsByClassName("deleteButton")[0];
+const deleteTaskButton = document.getElementsByClassName('deleteButton')[1];
```

Comment:
it's bad practice to do this by index number. what if you want to add another thing with that class? Then you'd have to change every index number.

Instead, give descriptive names.
`deleteButton--project`
`deleteButton--task`

Reply:
'it's bad practice to do this by index number. what if you want to add another thing with that class? Then you'd have to change every index number.' is this regarding to the (var projectNumber)? if so, how do I fix it?